modules/network/networker.go

```diff
@@ -188,6 +189,72 @@ func (n *networker) Join(member string, id modules.NetID) (join modules.Member,
 	return join, bridge.AttachNic(hostVeth, br)
 }
 
+// ZDBPrepare sends a macvlan interface into the
+// network namespace of a ZDB container
+func (n networker) ZDBPrepare() (string, error) {
```

Comment:
Can we have this with a more generic name. Networker should not care if this is ZDB or not, the provisiond instead (which knows the available modules) can instead use the generic functionality of both networker and storeaged to do the setup.

Reply:
This one is a bit more tricky to make generic cause it does really specific things for how 0-db networking is done at the moment.
Maybe I could split it into 3 or 4 other methods... I'll see what I can do. If that's too much work I'll just open a issue to follow up on that so we don't block this PR